server/service/src/preference/preferences/mod.rs

```diff
@@ -1,18 +1,35 @@
-use repository::{
-    EqualFilter, PreferenceFilter, PreferenceRepository, RepositoryError, StorageConnection,
-};
-use serde::de::DeserializeOwned;
+use repository::RepositoryError;
+
+pub mod types;
+pub use types::*;
 
 pub mod complex_pref;
 use complex_pref::*;
-mod use_payments_in_prescriptions;
-use use_payments_in_prescriptions::*;
+mod months_of_stock;
+use months_of_stock::*;
+mod show_contact_tracing;
+use show_contact_tracing::*;
 
 use crate::service_provider::ServiceContext;
 
+struct PreferenceRegistry {
+    pub show_contact_tracing: ShowContactTracing,
+    pub complex_one: ComplexOne,
+    pub months_of_stock: MonthsOfStock,
+}
+
+fn get_preference_registry() -> PreferenceRegistry {
+    PreferenceRegistry {
+        show_contact_tracing: ShowContactTracing,
+        complex_one: ComplexOne,
+        months_of_stock: MonthsOfStock,
```

Comment:
Then you just add your new pref to the registry here, and the `Preferences` struct below